src/Guest.java

```diff
@@ -0,0 +1,30 @@
+public class Guest {
+    private String nr;
+    public static int counter;
+    public int positionOfGuest;
+
+    public Guest(String name) {
+        nr = name;
```

Comment:
Why nr? Why not name the instance variable just name?

Reply:
This is the standard in Java, that the instance variable name is the same as local variable name. For example: https://github.com/nngrz/Java/blob/main/src/Computers.java#L5